### ISSUE TITLE
support github flavor shortcode

### ISSUE THREAD

**Issue by gasolin** [rhysd/remark-emoji#19]
according https://emojipedia.org/man-walking/

`:walking_man:` in github can transform to 🚶‍♂️

but remark-emoji only support :man-walking: 

```
:man_walking: (Emojipedia)
:man-walking: (Slack)
:walking_man: (Github)
```

**Comment by rhysd** [comment]
This is bug of upstream library [node-emoji](https://github.com/omnidan/node-emoji). I'll make a PR there.

**Comment by rhysd** [comment]
https://github.com/omnidan/node-emoji/pull/112

**Comment by rhysd** [comment]
Meanwhile I added [a workaround](https://github.com/rhysd/remark-emoji/blob/ad89b8356c7358aa63dadaeb5336afb488b5d0a5/index.js#L32-L47) for this and released new version v3.0.2. Please update dependency to the latest.

**Comment by gasolin** [comment]
@rhysd awesome, thanks!
